spec/linter-pep257-spec.coffee

```diff
@@ -0,0 +1,66 @@
+LinterPEP257 = require '../lib/init'
+path = require 'path'
+
+goodPath = path.join(__dirname, 'fixtures', 'good.py')
+badPath = path.join(__dirname, 'fixtures', 'bad.py')
+emptyPath = path.join(__dirname, 'fixtures', 'empty.py')
+
+describe "starts everything up", ->
+  lint = require('../lib/init').provideLinter().lint
+  beforeEach ->
+    waitsForPromise ->
+      atom.packages.activatePackage('linter-pep257')
+    waitsForPromise ->
```

Comment:
Not sure what to make of this, since I don't find any `await` in the `linter-eslint` specs. I'd suggest to leave it like this for now, merge it, and invite you guys to open further pull requests for improvements!